shared/util/kbfs.js

```diff
@@ -1,14 +1,34 @@
 /* @flow */
+import type {UserList} from '../common-adapters/usernames'
 
-// Parses the folder name and returns an array of usernames (TODO: handle read only-ers)
-export function parseFolderNameToUsers (folderName: string): Array<string> {
-  return folderName.split(',')
+// Parses the folder name and returns an array of usernames
+export function parseFolderNameToUsers (yourUsername: ?string, folderName: string): UserList {
+  const [rwers, readers = ''] = folderName.split('#')
+  return rwers.split(',')
```

Comment:
:sheep: Might be a little more readable to break this down into `const` assignments for the individual array steps